ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldProcess.cpp

```diff
@@ -105,9 +105,9 @@ ThermoMechanicalPhaseFieldProcess<DisplacementDim>::getDOFTableByProcessID(
 template <int DisplacementDim>
 void ThermoMechanicalPhaseFieldProcess<DisplacementDim>::constructDofTable()
 {
-    // Create single component dof in every of the mesh's nodes.
-    _mesh_subset_all_nodes =
-        std::make_unique<MeshLib::MeshSubset>(_mesh, _mesh.getNodes());
+    // For displacement equation.
+    constructDofTableOfSpecifiedProsessStaggerdScheme(
+        _mechanics_related_process_id);
```

Comment:
```suggestion
        _mechanics_process_id);
```

Reply:
This is an existed member. Its name change can be in another PR.